hermes/bin/src/runtime_extensions/wasi/cli/host.rs

```diff
@@ -33,20 +33,26 @@ impl cli::environment::Host for HermesRuntimeContext {
     }
 }
 
+impl cli::exit::Host for HermesRuntimeContext {
+    fn exit(&mut self, _status: Result<(), ()>) -> wasmtime::Result<()> {
+        Ok(())
+    }
+}
+
 impl cli::stdin::Host for HermesRuntimeContext {
     fn get_stdin(&mut self) -> wasmtime::Result<wasmtime::component::Resource<InputStream>> {
-        todo!()
+        Ok(wasmtime::component::Resource::new_own(0))
     }
 }
 
 impl cli::stdout::Host for HermesRuntimeContext {
     fn get_stdout(&mut self) -> wasmtime::Result<wasmtime::component::Resource<OutputStream>> {
-        todo!()
+        Ok(wasmtime::component::Resource::new_own(0))
```

Comment:
These don't need a log, because ultimately i would like to plumb stdout and stderr to our logging infrastructure for the wasm modules.  But they do need TODO comments saying that stdout/stderr needs to be redirected to our logs for an app.